docs/reference/mapping.asciidoc

```diff
@@ -68,6 +68,14 @@ via the <<multi-fields>> parameter.
 [float]
 === Settings to prevent mappings explosion
 
+Defining too many fields in an index or having too many nested documents or too many fields that contain nested
+documents are conditions that can lead to a mapping explosion, a case that occur when a cluster run ot of memory
```

Comment:
I would rather say: "conditions that can lead to a mapping explosion, which can cause out of memory errors and difficult situations to recover from.